Dockerfile

```diff
@@ -14,6 +14,12 @@ FROM registry.access.redhat.com/ubi8-minimal:8.4-200 as builder
 RUN microdnf install -y golang unzip && \
     go version
 
+# get restic. Needed for backup / restore capabilities
+ENV RESTIC_VERSION=0.12.0
+RUN microdnf install -y bzip2 && \
+    curl -sLO https://github.com/restic/restic/releases/download/v${RESTIC_VERSION}/restic_${RESTIC_VERSION}_linux_amd64.bz2 && \
+    bzip2 -d restic_${RESTIC_VERSION}_linux_amd64.bz2 && mv restic_* /tmp/restic && chmod +x /tmp/restic
```

Comment:
Should we care about architecture ?
Does licence allow to include resting in an image?

Reply:
Thanks, will build the link depending on the container arhitecture